python/cugraph-dgl/tests/test_cugraph_storage.py

```diff
@@ -18,14 +18,15 @@
 except ModuleNotFoundError:
     pytest.skip("cugraph_dgl not available", allow_module_level=True)
 
-import dgl
-import torch as th
+from cugraph.utilities.utils import import_optional
 import cudf
 import numpy as np
-
 from cugraph_dgl import CuGraphStorage
 from .utils import assert_same_sampling_len
 
+th = import_optional("torch")
+dgl = import_optional("dgl")
```

Comment:
If torch and/or dgl are not available, all test code that accesses `th` and/or `dgl` will have an uncaught RuntimeError, which will fail the test. Is that expected, or should those tests be skipped?

Reply:
I expect the tests to fail